docs/overview/sapi.rst

```diff
@@ -1,62 +1,231 @@
+        * - :ref:`Configuration file <sdk_index_cloud>`
+          - Configure using the :ref:`dwave CLI <dwave_cli>` tool or edit the file
+            manually.
+          - You can override this configuration by setting an environment
+            variable or explicitly in your code.
+        * - :std:doc:`Environment variables <oceandocs:docs_cloud/reference/configuration>`
+          - Configure :code:`DWAVE_API_TOKEN`. For example,
+            :code:`export DWAVE_API_REGION=ABC-1234 ... 789` in
+            a Unix shell.
+          - You can override this configuration explicitly in your code.
+        * - Explicit parameters
+          - You can set your API token directly in your code; for example,
+            :code:`sampler = LeapHybridCQMSampler(token="ABC-1234 ... 789")`
+          - Not recommended outside of testing (for security reasons).
+
+     .. note:: For non-Ocean clients, you set your token in the HTTP header
+        :code:`X-Auth-Token`; see the :std:doc:`sysdocs_gettingstarted:doc_rest_api`
+        guide for information.
+
+  .. [#]
+    When you work in D-Wave's `Leap <https://cloud.dwavesys.com/leap/>`_ IDE,
+    SAPI information such as your API token is pre-configured in the default
+    workspace's environment variables.
+
+* **Solver** (default: feature-based selection)
+
+  A D-Wave resource to be used to solve your submitted problems; for example, a
+  quantum-classical hybrid solver or an Advantage quantum computer.
+
+  .. dropdown:: Viewing Available Solvers
+
+     The :ref:`table_solvers_viewing` table shows various options to view
+     available solvers.
+
+     .. list-table:: Viewing Available Solvers
+        :name: table_solvers_viewing
+        :header-rows: 1
+
+        * - Where
+          - How
+          - Defaults
+          - Usage Notes
+        * - `Leap <https://cloud.dwavesys.com/leap/>`_ dashboard
+          - Log in to your Leap account.
+          - For users that belong to multiple projects, solvers accessible to the
+            current project are displayed; switch the current project if needed.
+          - Solvers may be local to a region.
+        * - :ref:`dwave-cloud-client <sdk_index_cloud>`
+          - Use the :ref:`dwave CLI <dwave_cli>` tool or Ocean's
+            :meth:`~dwave.cloud.client.Client.get_solvers` method.
+          - For users that belong to multiple projects, queries return solvers
+            accessible to the current API token; switch to another project's API
+            token if needed.
+          - Solvers may be local to a region; see the :ref:`sapi_intro_multiregion`
+            section to query solvers outside your default region.
+
+     Example: using :ref:`dwave CLI <dwave_cli>` to see the available solvers,
+     their parameters, and properties (the output shown below is illustrative only).
+
+     .. include:: ../docs_cli.rst
+        :start-after: cli-example-solvers-start-marker
+        :end-before: cli-example-solvers-end-marker
```

Comment:
I agree. I was tempted to replace this example imported from cloud-client with a `dwave solvers --al` example, and I do almost that in the "Example: Viewing a Region's Solvers" section. I suggest we update the example in cloud client and then it will be brought in here too